app/controllers/catalog_controller.rb

```diff
@@ -66,10 +85,14 @@ class CatalogController < ApplicationController
     config.add_facet_field 'contributor', field: 'cho_contributor_ssim', limit: true
     config.add_facet_field 'medium',      field: 'cho_medium_ssim', limit: true
     config.add_facet_field 'dc_rights',   field: 'cho_dc_rights_ssim', limit: true
-    config.add_facet_field 'holding_institution', field: 'agg_data_provider_ssim', limit: true
+    locale_encoded_fields.call('agg_data_provider').each do |code, field|
```

Comment:
@jacobthill is this okay in the interim for the facets? (You can see the before/after screenshots of the facets attached to this PR for a visual example).